content/events/net-superpowers-tour.mdx

```diff
@@ -137,10 +141,15 @@ _body:
           <VerticalListItem
             index={6}
             content={<>
-              Cloudy with a Chance of Mobile
+              Cloud Native Development
             </>}
             afterBody={<>
-              * Write once and run anywhere with .NET
+              Building a production-ready distributed application that is resilient, observable, and hosted in the cloud
+              
+              * DAPR
+              * .NET
```

Comment:
.NET Aspire is one thing -- not two :)

Reply:
> .NET Aspire is one thing -- not two :)
Done - merged into one